app/pulp/app/settings.py

```diff
@@ -112,7 +110,6 @@
     'DEFAULT_FILTER_BACKENDS': ('rest_framework.filters.DjangoFilterBackend',),
     'DEFAULT_PAGINATION_CLASS': 'pulp.app.pagination.UUIDPagination',
     'PAGE_SIZE': 100,
-    'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAuthenticated',)
```

Comment:
Yup. I usually keep this hunk in stash at all times because it's such a pain in the ass to work with auth on.